src/resolver.cpp

```diff
@@ -114,8 +114,7 @@ Ref locationInManagerContextForEntity(const openassetio::hostApi::ManagerPtr &ma
           // Error callback.
           // TODO(DF): Better conversion of BatchElementError to
           //  appropriate exception type.
-          std::string errorMsg = "OpenAssetIO error code ";
-          errorMsg += std::to_string(static_cast<int>(error.code));
```

Comment:
[Question] Why the removal of this, it seems like decent context?

Reply:
The new exception wrapper already adds an "OpenAssetIO error" prefix, so it seemed redundant.  I guess we can bring back the "error code" part if you think it's worthwhile?

Reply:
Reintroduced "error code" prefix in e362c5d9

Reply:
Ah, i didn't see you'd get an error : prefix. I agree that it was fine before, but it's equally fine now as well, so whatever's least work. Sorry for the runaround.